CMakePresets.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": 4,
+    "version": 3,
```

Comment:
This change is unrelated to the Create Uninitialized methods and because the CMake Presets uses the "include" feature, the [version must be 4 or above](https://cmake.org/cmake/help/latest/release/3.23.html#presets)